odoo/models.py

```diff
@@ -4833,7 +4833,7 @@ def create(self, vals_list):
                 # If a field is not stored, its inverse method will probably
                 # write on its dependencies, which will invalidate the field on
                 # all records. We therefore inverse the field record by record.
-                if all(field.store or field.company_dependent for field in fields):
+                if all(field.store or field.company_dependent or not field._depends for field in fields):
```

Comment:
@HydrionBurst the `company_dependent` case here probably needs to be removed in your jsonb PR :smile: